taxonomies/categories.txt

```diff
@@ -30379,7 +30379,7 @@ nl:Geraspte kzen
 
 <en:French cheeses
 fr:Maroilles, Marolles
-en:Maroilles 
+en:Maroilles
```

Comment:
en: should be first

Reply:
Thanks for the heads up, I will fix in the next commit